calculator.go

```diff
@@ -1,13 +1,88 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(a, b, c float64) float64 {
```

Comment:
What could we do here if we wanted to take _any_ number of extra arguments? Any ideas?

Reply:
We could use a variadic function.